tests/Doctrine/Tests/DBAL/Query/Expression/CompositeExpressionTest.php

```diff
@@ -18,6 +18,10 @@ public function testCount()
         $expr->add('u.group_id = 2');
 
         $this->assertEquals(2, count($expr));
+
+        $expr->add(new CompositeExpression(CompositeExpression::TYPE_OR, []));
+
+        $this->assertEquals(2, count($expr));
```

Comment:
Could you move it to a separate test method, and verify both paths then? Specifically:

 * When an empty compositeexpression is added
 * When a non-empty compositeexpression is added